packages/nextjs/src/__tests__/telemetry-collector.test.ts

```
@@ -0,0 +1,84 @@
+          isEnabled = true;
+          isDebug = false;
+          record(ev: any) {
+            records.push(ev);
+          }
+        },
+      };
+    });
+
+    // Re-require after mock
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const { NextJSTelemetryCollector: Collector } = require('../utils/telemetry-collector');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)